etc/systemd/system/greenboot-healthcheck.service

```diff
@@ -0,0 +1,25 @@
+#  SPDX-License-Identifier: LGPL-2.1+
+#
+#  This file is part of greenboot.
+#
+#  greenboot is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+[Unit]
+Description=greenboot Health Checks Runner
+Before=boot-complete.target
+After=network-online.target
+Wants=network-online.target
+OnFailure=redboot.target
+OnFailureJobMode=fail
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+ExecStart=/usr/libexec/greenboot/greenboot check
+
+[Install]
+RequiredBy=boot-complete.target
+WantedBy=multi-user.target
```

Comment:
this should be a dropin unit at e.g. path `/usr/lib/systemd/system/greenboot-healthcheck.service.d/10-network-online.conf` containing only this:
```
[Unit]
After=network-online.target
Wants=network-online.target
```